<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main_window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="layout_global">
      <item>
       <widget class="QTextEdit" name="text_edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border : none;
padding : 10px;
background : white;</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.AppleSystemUIFont';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="layout_edit">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item row="7" column="2">
         <widget class="QPushButton" name="button_align_right">
          <property name="text">
           <string>R</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="QComboBox" name="combo_font_family">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <item>
           <property name="text">
            <string>Helvetica Neue</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Baskervill</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Andale Mono</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hiragino Sans CNS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Times New Roman</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gill Sans</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="button_bold">
          <property name="styleSheet">
           <string notr="true">font-weight : bold;</string>
          </property>
          <property name="text">
           <string>B</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="button_italic">
          <property name="styleSheet">
           <string notr="true">font-style : italic;</string>
          </property>
          <property name="text">
           <string>I</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QLabel" name="label_font">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">
</string>
          </property>
          <property name="text">
           <string>Font</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="button_underline">
          <property name="styleSheet">
           <string notr="true"> text-decoration : underline;</string>
          </property>
          <property name="text">
           <string>U</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QComboBox" name="combo_font_type">
          <item>
           <property name="text">
            <string>Regular</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Italic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Semibold</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Semibold Italic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bold</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bold Italic</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QPushButton" name="button_align_justify">
          <property name="text">
           <string>J</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="button_align_center">
          <property name="text">
           <string>C</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="button_color">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton	
{
	padding : 2px 11px 1px 11px;
	border-top : 1px solid #c8c8c8;
	border-bottom : 1px solid #acacac;
	border-left : 1px solid #c2c2c2;
	border-right : 1px solid #c2c2c2;
	margin : 2px 0px 1px 0px;
	background-color : black;
	border-radius : 4px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="3">
         <widget class="QLabel" name="label_color">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Text color</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="button_crossed">
          <property name="styleSheet">
           <string notr="true"> text-decoration : line-through;</string>
          </property>
          <property name="text">
           <string>S</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QSpinBox" name="spin_font">
          <property name="value">
           <number>11</number>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="button_align_left">
          <property name="text">
           <string>L</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="4">
         <widget class="QLabel" name="label_alignment">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Alignment</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>576</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
